src/umich/general/InputFile.h

```diff
@@ -80,7 +80,7 @@ class InputFile
     /// Any previous values in the buffer will be deleted.
     /// \param bufferSize number of bytes to read/buffer at a time,
     /// turn off read buffering by setting bufferSize = 1;
-    inline void bufferReads(unsigned int bufferSize = DEFAULT_BUFFER_SIZE)
+    inline void bufferReads(int bufferSize = DEFAULT_BUFFER_SIZE)
```

Comment:
@changebio why do we have to change all unsiged int to int?

Reply:
There are some compiling errors because of unsigned type. https://stackoverflow.com/questions/40269739/c-ambiguous-call-to-overloaded-function-with-unsigned-int